html/datasets.html

```diff
@@ -0,0 +1,311 @@
+                    */
+                    {'field': 'mostSpecificType', 'display': 'Type'},
+                    {'field': 'publicationYear', 'display': 'Publication Year'},
+                    {'field': 'authors.name.exact', 'size': 20, 'display': 'Author'},
+                    {'field': 'authors.organization.name.exact', 'display': 'Author Organization'},
+                    {'field': 'dcoCommunities.name.exact', 'display': 'DCO Community'},
+                    {'field': 'portalGroups.name.exact', 'display': 'Portal Group'},
+                    {'field': 'projects.name.exact', 'display': 'Project'},
+                    {'field': 'dataTypes.name.exact', 'display': 'Data Type'}
+                ],
+                search_sortby: [
+                    {'display':'Year','field':'publicationYear'},
+                    {'display':'Title','field':'title'}
+                ],
+                render_result_record: function(options, record)
+                {/*
+                 Contents to be displayed in individual records.
+                 Here is a template below. (Not real)
+                 ****************************************************************************
+                 * (The title; with link if dataset has DOI) (Publication Year)
+                 * Type: Dataset
+                 * DCO-ID: (e.g.) 11121/8158-6700-3912-5989-CC (linked to metadata in vivo)
+                 * DCO Communites: Data Science Team; Extreme Physics and Chemistry Community; ...
+                 * DCO Portal Groups: Data Science Community; ...
+                 * Authors: Ma, Xiaogang (Marshall); Ghiorso, Mark S.; Zhong, Hao; ...
+                 * Projects: DECADE - CO2 Soil Degassing at Piton de la Fournaise; ...
+                 * Has Data Types: Thermodynamics of chemicals and minerals; ...
+                 * Citations (Source Papers): Inverted High-Temperature Quartz, Unit Cell Parameters ...
+                 * Distributions: (Note: Some datasets have multiple distributions; some distributions have multiple files.)
+                 *     20150827-0138 (Direct access: GhiorsoEtAl1979)
+                 *     Han's Queries (Oct 2013) (Direct access: EPC Queries; DS Queries; DL Queries; DE Queries; README; RF Queries)
+                 *     ...
+                 * [Badges of doi and metadata with link][]..
+                 ****************************************************************************
+                 Below are implementations of each item.
+                */
+                    var doiUrl = "https://dx.doi.org/"+record["doi"];
+
+                    var html = "<tr><td>";
+
+                    // Show DCO logo as datasets are all DCO publications!
+                    // We can also uncommented the if-condition if we want to find out which datasets are not registered as
+                    // DCO publications, as they should be.
+//                    if (record["isDcoPublication"])
```

Comment:
delete the isDcoPublication code? Don't just comment it out. And we don't need to display the logo for each dataset. Remove that as well.